backend/fms_core/filters.py

```diff
@@ -213,4 +215,15 @@ def quantity_ng_gte_filter(self, queryset, name, value):
 
     class Meta:
         model = SampleNextStep
-        fields = _sample_next_step_filterset_fields
\ No newline at end of file
+        fields = _sample_next_step_filterset_fields
+
+class ReadsetFilter(GenericFilter):
+    id__not_in = django_filters.CharFilter(field_name='id', method='filter_id_not_in')
```

Comment:
This is an oddly specific filter. What is the use case related to this ?

Reply:
@ckostiw can elaborate on this

But the reason why i've introduced it is so that we can avoid pulling in hundreds of readsets that the users don't want to see, and the Readset viewset for some reason did not support id__not_in